packages/router/router.ts

```diff
@@ -339,6 +339,23 @@ export type HydrationState = Partial<
   Pick<RouterState, "loaderData" | "actionData" | "errors">
 >;
 
+export interface DefaultCallLoaderOrActionFunction {
+  (
+    type: "loader" | "action",
```

Comment:
Does this need to call actions?  I've been envisioning this as a loader-only concept